src/api.js

```diff
@@ -7,3 +7,11 @@ export const getAllArticles = async () => {
 
   return response.data;
 };
+
```

Comment:
I'd consider using axios.create() here so you will have a base URL which can be used for all your request functions. https://notes.northcoders.com/courses/js-front-end/axios#axios-instance